src/controllers/MusicController.ts

```diff
@@ -201,6 +205,13 @@ export default class MusicController extends Controller {
 		if (this.state.state === "playing") this.play(true);
 	};
 
+	nowPlaying = () => {
```

Comment:
This function seems useless. There's a `state` getter that you can use instead. This function is too unreliable. You could move the logic to the command easily.